components/progress-bar/script.js

```diff
@@ -10,6 +10,24 @@ import template from './template.html';
 
 const BaseComponent = components.BaseComponent;
 
+/**
+ * Check if a value is a number and if not - log an error
+ * @param {string} propName - the name of the property that needs to be validated
+ * @param {any} value
+ * @returns {boolean} - true if it is a number or a string that can be cast to number, false if not
+ */
+function isAPositiveNumberValidation(propName, value) {
```

Comment:
```suggestion
function isNumberPositive(propName, value) {
```
or `isNumberPositiveValidation` may be a more appropriate name for this method but don't mind it if you don't like my suggestion as it is my personal vision :D

Reply:
I agree with you - changed.